locustempus/templates/main/course_edit.html

```diff
@@ -14,8 +16,17 @@
            <input data-cy="field-workspace-title" type="text" name="title" id="id_title" value="{{ object.title }}" required="required" class="form-control {% if form.title.errors %}is-invalid{% endif %}" />
         </div>
         <div class="form-group">
-            <label for="id_description">Description:</label>  {# not hooked up #}
-            <textarea name="description" cols="40" rows="3" class="form-control" maxlength="200" placeholder="A brief summary of your workspace. NOT HOOKED UP YET" id="id_description"></textarea>
+            <label for="{{ form.description.id_for_label }}">Description:</label>
+            {% if form.description.errors %}
+                {{ form.description|add_class:"form-control is-invalid"|attr:"data-cy:workspace-description" }}
+                <div class="invalid-feedback">
+                    {% for error in form.description.errors %}
+                        {{ error }}
+                    {% endfor %}
+                </div>
+            {% else %}
+                <textarea name="id_description" cols="40" rows="3" class="form-control" maxlength="200" placeholder="A brief summary of your workspace." id="id_description" data-cy="field-workspace-description">{{ object | course_description }}</textarea>
```

Comment:
The course description does not appear to be saving. To reproduce: Go to Edit Course, change the Description, hit save. (Navigate back to the workspace detail page.) Go to Edit Course -- the description was not saved.

Reply:
Note: See the comment above about the `id_description`. Fix that name field should fix this.